harold/plugins/deploy.py

```diff
@@ -29,6 +33,9 @@
 
 class DeployConfig(PluginConfig):
     organizations = Option(tup)
+    default_hours_start = Option(str)
```

Comment:
You can put `parse_time` and `pytz.timezone` in the `Option` in place of `str` to have the parsing done up-front during startup and let the code in the application itself be a bit terser.